include/message_filters/subscriber.h

```diff
@@ -140,9 +181,24 @@ class Subscriber : public SubscriberBase, public SimpleFilter<M>
    *
    * \param node The rclcpp::Node::SharedPtr to use to subscribe.
    * \param topic The topic to subscribe to.
-   * \param qos (optional) The rmw qos profile to use to subscribe
+   * \param qos (optional) The qos profile to use to subscribe
    */
   void subscribe(rclcpp::Node::SharedPtr node, const std::string& topic, const rmw_qos_profile_t qos = rmw_qos_profile_default)
+  {
+    rclcpp::QoS rclcpp_qos(rclcpp::QoSInitialization::from_rmw(qos));
+    subscribe(node, topic, rclcpp_qos);
+  }
+
+  /**
+   * \brief Subscribe to a topic.
+   *
+   * If this Subscriber is already subscribed to a topic, this function will first unsubscribe.
+   *
+   * \param node The rclcpp::Node::SharedPtr to use to subscribe.
+   * \param topic The topic to subscribe to.
+   * \param qos (optional) The qos profile to use to subscribe
```

Comment:
According to the signature it is not optional.

```suggestion
   * \param qos The qos profile to use to subscribe
```

Reply:
Fixed